src/components/Header.js

```diff
@@ -25,6 +27,13 @@ const Header = React.memo(function Header() {
           align-items: center;
         `}
       >
+        <img
+          src={headerLogoSvg}
+          height={6 * GU}
```

Comment:
Should we set the `width` instead? That way, the title next to it wouldn’t jump after the image loads.